### PR TITLE
Select and create Account type

### PR DESCRIPTION
**Pull request by autotelik**
Implements basic models to mirror CRM
Static pages in place
Journey first page implemented

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -28,8 +28,9 @@ gem "dotenv-rails", "~> 2.1"
 
 gem 'copy_kit', git: 'https://github.com/EnvironmentAgency/copykit'
 gem 'datashift',  git: 'https://github.com/autotelik/datashift' #path: '../../github/datashift'
-gem 'datashift_journey',git: 'https://github.com/autotelik/datashift_journey'
-# path: '../datashift_journey'
+gem 'datashift_journey',
+    #git: 'https://github.com/autotelik/datashift_journey'
+ path: '../datashift_journey'
```
Comment (lewispb) [inline Gemfile]: Did you forget to switch this back?

File: app/forms/account_type_form.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    new(application.account, application)
+  end
+
+  def self.params_key
+    :account_type
+  end
+
+  property :account_type, virtual: true
+
+  validates :account_type,  presence: { message: I18n.t("#{locale_errors}.account_type.blank") }
```
Comment (lewispb) [inline app/forms/account_type_form.rb]: Might be worth checking out the scope option of the `t` method, something like:

`I18n.t(".account_type.blank", scope: locale_errors)`

File: app/models/application.rb [hunk 3]
```diff
@@ -2,14 +2,16 @@ class Application < ActiveRecord::Base
...
-
-  belongs_to :operator , class_name: "Contact"
+  has_many :application_permits, foreign_key: :application_id, dependent: :restrict_with_exception
+
+  has_many :permits, through: :application_permits, dependent: :restrict_with_exception
+
+  belongs_to :operator ,            class_name: "Contact"
+  belongs_to :finance_contact ,     class_name: "Contact"
+  belongs_to :application_contact , class_name: "Contact"
+  belongs_to :application_contact , class_name: "Contact"
```
Comment (lewispb) [inline app/models/application.rb]: Duplicate here, should it be `operational_contact`?

File: Gemfile (hunk 1 above)
Comment (autotelik) [inline Gemfile]: errr this code is in development. we create a PR as soon as we push. doesn't mean it's ready for review

**Comment by autotelik** [comment]
:eyes:

**Comment by lewispb** [comment]
:ship:

**Pull request merged**